Extension/package.nls.json

```diff
@@ -400,5 +400,26 @@
     "c_cpp.debuggers.logging.category.verbose.description": "Logs that are used for interactive investigation during development. These logs should primarily contain information useful for debugging and have no long-term value.",
     "c_cpp.debuggers.logging.category.warning.description": "Logs that highlight an abnormal or unexpected event in the application flow, but do not otherwise cause the application execution to stop.",
     "c_cpp.debuggers.logging.category.error.description": "Logs that highlight when the current flow of execution is stopped due to a failure. These should indicate a failure in the current activity, not an application-wide failure.",
-    "c_cpp.debuggers.logging.category.none.description": "Not used for writing log messages. Specifies that a logging category should not write any messages."
+    "c_cpp.debuggers.logging.category.none.description": "Not used for writing log messages. Specifies that a logging category should not write any messages.",
+    "c_cpp.walkthrough.title": "Get Started with C++ Development",
+    "c_cpp.walkthrough.description": "Dive into VS Code's rich C++ development experience.",
+    "c_cpp.walkthrough.set.up.title": "Set up your C++ Environment",
+    "c_cpp.walkthrough.activating.description": "Activating the C++ Extension to determine whether your C++ Environment has been set up.\nActivating Extension...",
+    "c_cpp.walkthrough.no.compilers.description": "We could not find a C++ compiler on your machine, which is required to use the C++ Extension. Follow the instructions on the right to install one, then click “Find my new compiler” below.\n[Find my new compiler](command:C_Cpp.RescanCompilers)",
+    "c_cpp.walkthrough.compilers.found.description": "The C++ extension requires a compiler to be installed. Select one from those already on your machine by clicking the button below.\n[Select your default Compiler](command:C_Cpp.SelectDefaultCompiler)",
```

Comment:
Why is Compiler capitalized?

Reply:
It's a link displayed as a button.  Like a title, it might make sense to capitalize words on a button.  Possibly 'default' should be capitalized as well.

Reply:
Can you make it title cased then?